README.md

```diff
@@ -59,17 +59,19 @@ See the command line options with:
    
 ### DNS records
 
-The feed-dns controller assumes that it can overwrite any entry in the supplied DNS zone and manages ALIAS
+The feed-dns controller assumes that it can overwrite any entry in the supplied DNS zone and manages ALIAS and CNAME
 records per ingress.
 
 On startup, all ingress entries are queried and compared to all the Record Sets in the configured hosted zone.
 
-Any ALIAS records pointing to one of the ELBs associated with this controller that do not have an ingress
-entry are deleted. For any new ingress entry, an ALIAS record is created to point to the correct ELB. Existing
```

Comment:
I don't think we should delete this part, as these details are pretty important (and just got added in a recent PR). We should mention the "cname" behaviour separately I think.

Reply:
I haven't deleted this part completely; I've just swapped out the AWS specific terms for generic ones because the behaviour is the same whether you're talking ELBs or non-ELBs. At the bottom, I try to make it clear that the record types are different depending on whether an ELB is used or not.

But if that's still confusing then I'm happy to go back and add another paragraph for non-ELBs.